lms/djangoapps/course_blocks/transformers/tests/test_library_content.py

```diff
@@ -70,7 +70,6 @@ def get_course_hierarchy(self):
                                     '#ref': 'vertical1',
                                     '#children': [
                                         {
-                                            'metadata': {'category': 'library_content'},
```

Comment:
So `metadata` is an old thing - it gets up-leveled as just straight properties on the xblock nowadays. So that line could be equivalently written as:
```
                                     '#children': [
                                        {
                                            'category': 'library_content',
```

Without the metadata bit at all. And the reason I took it out is because it was trying to set `category` as a field item. Which it isn't, it's a separate thing (not a field, but a class signifier).

In Old Mongo, you can set anything you want on an xblock. So this was setting a dynamic python property on the xblock called `category`, which is harmless but also pointless.

Split modulestore is a little fussier, and was erroring out here that `category` was not a field, and rightly so.